apps/src/netsim/NetSimRouterLogModal.html.ejs

```diff
@@ -21,6 +21,21 @@ var showPacketInfo = headerFields.indexOf(Packet.HeaderType.PACKET_INDEX) > -1 &
  */
 
 /**
+ * @name canToggleRouterLogMode
+ * @type {boolean}
+ */
+
+/**
+ * @name currentTrafficFilter
```

Comment:
are the options for this clearly enumerated anywhere? It looks like they are independently implemented in a couple of different places